helper.sh

```diff
@@ -7,6 +7,7 @@ deploy_to_env () {
     echo Deploying to ${env}
     ssh -o "StrictHostKeyChecking no" -t ec2-user@13.239.37.152 "rm -rf /ecs/cp3402-$env-storage/wp-content/plugins"
     ssh -o "StrictHostKeyChecking no" -t ec2-user@13.239.37.152 "rm -rf /ecs/cp3402-$env-storage/wp-content/theme/custom_theme"
+    ssh -o "StrictHostKeyChecking no" -t ec2-user@13.239.37.152 "rm -rf /ecs/cp3402-$env-storage/wp-content/uploads"
     scp -r -o "StrictHostKeyChecking no" ./wp-content/plugins/. ec2-user@13.239.37.152:/ecs/cp3402-${env}-storage/wp-content/plugins/
     scp -r -o "StrictHostKeyChecking no" ./wp-content/themes/custom_theme/. ec2-user@13.239.37.152:/ecs/cp3402-${env}-storage/wp-content/themes/custom_theme/
```

Comment:
Need to copy this line too. First line removes existing files, and the the next replaces them

Reply:
I don't understand. I did copy it?

Reply:
Issue has been resolved